### PR TITLE
Added Update Progress helper methods.

### PR DESCRIPTION
**Pull request by droyad**
Resolves https://github.com/OctopusDeploy/Issues/issues/5696

### CODE REVIEW

File: source/Calamari.Shared/Integration/Scripting/Bash/Bootstrap.sh [hunk 1]
```diff
@@ -90,8 +90,8 @@ function set_octopusvariable
 		MESSAGE="$MESSAGE value='$(encode_servicemessagevalue "$2")'"
 	fi
 	
-	if [ ! -z "${3:-}" ] && [ "$3" = "-sensitive" ]
-	then
+	if [ ! -z "${3:-}" ] && [ "$3" = "-sensitive" ]
```
Comment (droyad) [inline source/Calamari.Shared/Integration/Scripting/Bash/Bootstrap.sh]: I think this is due to Rider normalising line endings

File: source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1 [hunk 2]
```diff
@@ -356,6 +356,21 @@ function New-OctopusArtifact
     }
 }
 
+function Update-Progress
+{
+	[CmdletBinding()]
+	param(
+        [int] $percentage, 
+        [Parameter(ValueFromPipeline=$true)][string]$message
```
Comment (michaelnoonan) [inline source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1]: I wouldn't have picked the message as the `ValueFromPipeline`, but I also can't think of a reason not to pick the message.

Comment (droyad) [reply source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1]: Do yo think we should swap the parameters around? `Update-Process "Hello World" 50`

Comment (michaelnoonan) [reply source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1]: No, I like having the percentage first. :+1: I think it's good for go.

**Pull request merged**